test/codeBlock/appmap.yml

```diff
@@ -1,2 +1,8 @@
-name: codeBlock
-appmap_dir: tmp/appmap
\ No newline at end of file
+name: appmap-node
+language: javascript
+appmap_dir: tmp/appmap
+packages:
+  - path: .
+    exclude:
+      - node_modules
```

Comment:
Because appmap-node always puts this exclude entry in the generated config file, you could consider testing for the presence of “node_modules” in the config file string before performing the language fix up. Probably not needed but would add a bit of protection.

Reply:
It's only setting `language` if it's currently missing. Do you think we need to go beyond that?